### ISSUE TITLE
SophiApp thinks Windows 11 LTSC is an outdated insider build.

### ISSUE THREAD

**Issue by sdhEmily** [Sophia-Community/SophiApp#207]
### Prerequisites

- [X] Verify that this is not a Windows issue;
- [X] Refer to the [System requirements](https://github.com/Sophia-Community/SophiApp/blob/master/README.md#system-requirements);
- [X] I do not use a homebrew Windows image;
- [X] I did not tweak Windows before that could cause system instability;
- [X] Attach a created log. You may find it in the `Logs` folder.

### Steps To Reproduce

1. Use Windows 11 LTSC
2. Open SophiApp

### Expected behavior

SophiApp starts on my up to date version of LTSC 11

### Actual behaviour

SophiApp thinks my version is an outdated insider build and refuses to start.

### Windows Version

Windows 10 IoT Enterprise LTSC 2024 24H2 build 26100.1150

### SophiApp version

1.0.97.0

### SophiApp release type

SophiApp

### Logs

Windows 10 IoT Enterprise LTSC 2024 24H2 build 26100.1150
Computer name: ZENITH
User: Emily
User domain: ZENITH
User culture: .
User region: .
App version: 1.0.97.0
App is release: True
App folder: "C:\Users\Emily\Downloads\SophiApp\"
App localization: EN
App theme: DARK

21.07.2024 00:37:40 Debug mode is: False
21.07.2024 00:37:40 Active view is: Loading
21.07.2024 00:37:40 Advanced settings is visible: False
21.07.2024 00:37:40 The "UWP for all users" switch state is: UNCHECKED
21.07.2024 00:37:40 The OS conditions check started
21.07.2024 00:37:40 OsVersionCondition run result: True
21.07.2024 00:37:40 The next condition to be run: OsBuildVersionCondition
21.07.2024 00:37:40 OsBuildVersionCondition run result: False
21.07.2024 00:37:40 The next condition to be run: OsFilesCorruptedCondition
21.07.2024 00:37:41 Active view is: OsBuildVersion
21.07.2024 00:37:41 It took 1 second(s) to check the OS conditions

**Comment by farag2** [comment]
Hello. As you may see commits, the active development of the 1.x version has been stopped. The LTSC supported will be added when 2.0 version will be released.

https://github.com/Sophia-Community/SophiApp?tab=readme-ov-file#sophiapp-20

**Comment by Elllkere** [comment]
If someone still need this i manually removed OS check, so it will always say good. I tested it with 24h2 which also cant be used by default.
[SophiApp.zip](https://github.com/user-attachments/files/17332726/SophiApp.zip)

**Comment by farag2** [comment]
I'd not advice to use it as it contains too many bugs. Please wait for the 2.0 release. Better to use Sophia Script. Hope, the first public beta (without all functions) will be dropped on the New Year eve.

**Comment by Elllkere** [comment]
> it contains too many bugs

yeah, i know what usually causes errors and avoid it and do it myself, also compiled 2.0 to use done functions instead of 1.0)

**Comment by dKenGuru** [comment]
So, where is beta?

**Comment by farag2** [comment]
On my PC. :) we are aimed on 25H1

**Comment by dKenGuru** [comment]
May soon. Still no real eta, no beta, no alpha.

**Comment by farag2** [comment]
It exists, but we have to work on the project after work. We aren't unemployed. 5oo many issues with WinUI 3. That's why we have to postpone it every time. But the last category remains. The end is near.

From April 11
![Image](https://github.com/user-attachments/assets/7e6667c9-3837-440e-a699-51458f044749)

**Comment by sdhEmily** [comment]
🫡

**Comment by dKenGuru** [comment]
10 days before H1 will end

**Comment by farag2** [comment]
> 10 days before H1 will end

We know. And the work is in process.

**Comment by dKenGuru** [comment]
25H2 уже вышла. А gui все нет

**Comment by farag2** [comment]
Если нет, то значит, ещё нет. Предлагаешь бросить работу, семью и тратить все свободное время на программу. :)
Коммиты делаются каждую неделю, как видишь. Значит, всё делается, и все будет.

**Comment by dKenGuru** [comment]
Да выкладывать хоть dev сборки. На свой страх и риск.

**Comment by farag2** [comment]
Ещё не время. Любой может уже скомпилировать текущий исходный код и получить то же самое.

**Comment by peschull** [comment]
Just adding another data point and a bit of clarification regarding LTSC here.

I’m seeing the same behavior on **Windows 11 IoT Enterprise LTSC 2024, version 24H2, build 26100.x** with **SophiApp 1.0.97.0** – the app treats the system as an “outdated insider build” and refuses to start (`OsBuildVersionCondition = False`).

However, this LTSC SKU is no longer an Insider build – it’s the **final RTM for Windows 11 IoT Enterprise LTSC 2024**, with a 10-year support lifecycle and monthly quality updates from Microsoft (same core as 24H2, build 26100). So from the OS side this is a fully supported, production build, not a preview.

I fully understand that active development of the 1.x line has stopped and that LTSC support is planned for 2.0 / WinUI 3. Still, for the long term:

* It might be worth treating 24H2 LTSC builds as **“supported but advanced users only”** instead of hard-blocking startup, or
* At least degrading the OS check to a **warning** with an opt-in “continue at your own risk” button.

That would make SophiApp usable on current LTSC devices without requiring patched binaries, while still making it clear that 1.x is in maintenance mode.

In the meantime, I’ll stick to **Sophia Script** on this machine as you suggested, and will happily switch to SophiApp 2.0 once there is a public build that supports LTSC officially.

Thanks for all the work you put into this project – it’s extremely useful for power users and admins.

**Comment by Inestic** [comment]
We are currently focused on fixing bugs and working on the appearance of version 2.0 (responsive design). Will there be anything for the LTSC version? Perhaps, but it's too early to say. You support is very motivating, and we will be open to ideas and suggestions after the stable version is released (as long as these ideas do not conflict with our project manager's vision 😄 ).

**Comment by farag2** [comment]
@peschull, thanks for the suggestion. Stay tuned. Hopefully, we will be able to show something to the community. :)

**Comment by farag2** [comment]
First step: https://github.com/Sophia-Community/SophiApp/commit/52c334c3c1433c05fdd6841bb772ba2ddcb436cc
